pricing-page-exercise/dilara/styles.css

```diff
@@ -0,0 +1,234 @@
+.nav-bar {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content:space-between;
+    min-height:38px;
+    padding: 8px 24px;
+    align-items: center;
+    border-bottom: 1px solid #dee2e6;
```

Comment:
I see you mixing all kinds of colors. Try to use HEX colors everywhere and RGBA when you need to have a color with opacity. This is what's used the most in real life.